src/pages/User/content/SpaceProfile.tsx

```diff
@@ -246,10 +205,19 @@ export const SpaceProfile = ({ user, docs }: IProps) => {
     user.location?.countryCode || user.country?.toLowerCase() || undefined
 
   return (
-    <ProfileWrapper mt={4} mb={6} data-cy="SpaceProfile">
-      <ProfileWrapperCarousel>
+    <Container
+      mt={4}
+      mb={6}
+      sx={{
+        border: '2px solid black',
+        borderRadius: '10px',
+        overflow: 'hidden',
+      }}
```

Comment:
Thanks for this Ben, only thing I've noticed is we need a max-width of 1000px to match the existing styling.

```suggestion
      sx={{
        border: '2px solid black',
        borderRadius: '10px',
        overflow: 'hidden',
        maxWidth: '1000px',
      }}
```